upgrade/Validate_CSM_Health_During_Upgrade.md

```diff
@@ -4,7 +4,7 @@
   since the CSM services were upgraded. This allows the various Kubernetes resources to
   initialize and start.
 - Although it is not recommended, the [Booting CSM `barebones` image](../operations/validate_csm_health.md#5-booting-csm-barebones-image)
-  test may be skipped if all compute nodes are active running application workloads.
+  test may be skipped if all compute nodes are active running application workloads. A [Known issue](../troubleshooting/known_issues/barebones_boot_test_failure.md) exists for CSM 1.6.0
```

Comment:
```suggestion
  test may be skipped if all compute nodes are active running application workloads. In CSM 1.6.0, a known issue may
  cause a [Barebones Boot Test Failure](../troubleshooting/known_issues/barebones_boot_test_failure.md).  
```

Reply:
If this always happens, then probably this should say "causes" instead of "may cause". I wasn't sure which was the case.